### PR TITLE
Webpack resolve alias support

### PR DESCRIPTION
**Pull request by peterhass**
Tested with nebenan-frontend (react config) and gracefultools (base config).

npm will complain about unsatisfied webpack peerDependency but it still works. Looked into `eslint-import-resolver-webpack`: It never loads anything from webpack package but future updates may break our solution here but even then, we could fix it by writing our own [resolver](https://github.com/benmosher/eslint-plugin-import/tree/master/resolvers/webpack)

### CODE REVIEW

File: javascript/packages/eslint-config-nebenan-base/index.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+const process = require('process');
+const path = require('path');
+
+const rootDir = process.cwd();
```
Comment (peterhass) [inline javascript/packages/eslint-config-nebenan-base/index.js:4]: Shitty solution - suggestions?

File: javascript/packages/eslint-config-nebenan-base/index.js [hunk 2]
```diff
@@ -11,6 +16,17 @@ module.exports = {
           '.es',
         ],
       },
+      webpack: {
+        config: {
+          resolve: {
+            extensions: ['.es', '.js', '.json'],
+            alias: {
+              '@': rootDir,
+              '@public': path.join(rootDir, 'public/'),
```
Comment (peterhass) [inline javascript/packages/eslint-config-nebenan-base/index.js:25]: TOOD: not needed

File: javascript/packages/eslint-config-nebenan-base/index.js (hunk 1 above)
Comment (peterhass) [reply javascript/packages/eslint-config-nebenan-base/index.js:4]: https://github.com/goodhood-eu/uni-config/blob/master/index.js#L16

**Comment by suprMax** [comment]
@peterhass write the rules for using this alias also

File: javascript/README.md [hunk 3]
```diff
@@ -1359,6 +1359,30 @@ Inspired by: [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascrip
     import barCss from 'bar.css';
     ```
 
+  <a name="modules--absolute-imports"></a>
```
Comment (peterhass) [inline javascript/README.md:1362]: - Change to recommendation
- "Use absolute if too long"

**Pull request merged**